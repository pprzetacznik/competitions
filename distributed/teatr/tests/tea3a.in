80000000 0 -1 -1 1000000 6881491782827497215
