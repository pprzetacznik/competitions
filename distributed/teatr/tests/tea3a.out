1599997734247474
